source/functions/orbital_velocity.m

```diff
@@ -1,5 +1,5 @@
 function [u,v,w,xa,ya,za] = orbital_velocity(disp,k,omega,waveAmp,theta,t,rampT, beta, h, g)
```

Comment:
@akeeste I'm struggling to find where this function is used within WEC-Sim. Although, we need to calculate the flow velocity and acceleration to calculate the Morison Element forces, we do this in the ME block and do not reference a separate function. Trying to determine if this is perhaps an outdated function that could be removed?

Reply:
I just looked at the orbital velocity calculation in the ME block, it looks like an updated / rewritten version of the stand-alone function. It seems that it's likely outdated and not used

Reply:
Thanks, that was my initial conclusion after reviewing this file and noticing this is not called in the ME block. If no one else on the WEC-Sim team has a reason to keep this function then suggest removing before pushing to the dev branch.

Reply:
@nathanmtom I agree